packages/core/package.json

```diff
@@ -23,27 +23,18 @@
     "!**/__mocks__"
   ],
   "scripts": {
-    "android": "react-native run-android",
-    "postinstall": "npx patch-package",
-    "ios": "npx react-native run-ios --device",
-    "lint": "eslint --color .",
-    "build": "bob build",
-    "prepack": "bob build",
-    "pre-commit-lint": "npx lint-staged",
-    "prettier": "prettier --ignore-path .gitignore '**/*.+(js|jsx|json|ts|tsx|md|yml|yaml)'",
-    "setup": "cd .. && husky install .husky",
-    "start": "npx react-native start --reset-cache",
-    "style-check": "npm run prettier -- --check",
-    "style-format": "npm run prettier -- --write",
-    "style-watch": "onchange '**/*.+(js|jsx|json|ts|tsx|md|yml|yaml)' -- prettier --write {{changed}}",
+    "build": "echo 'building core 🏗️' && yarn clean && bob build && echo 'core built ✅'",
+    "clean": "rimraf ./lib",
     "test": "TZ=GMT jest",
-    "test:coverage": "TZ=GMT jest --coverage",
+    "coverage": "TZ=GMT jest --coverage",
     "test:watch": "TZ=GMT jest --watch --notify --notifyMode=change --coverage",
     "typecheck": "tsc --noEmit"
   },
   "devDependencies": {
     "@babel/core": "^7.20.0",
     "@babel/runtime": "^7.20.0",
+    "@bifold/oca": "0.0.0",
```

Comment:
Made these 0.0.0 so that the next major version it will be bumped to would be 1.0.0 (for the first release of the new packages)